libflux/flux-core/Cargo.toml

```diff
@@ -13,6 +13,7 @@ default = ["strict"]
 strict = []
 
 [dependencies]
+comrak = "0.10.1"
```

Comment:
I'm not sure that we should be putting this work in `flux-core`. I'd much rather see it in the `flux` crate